docs/rdf-terminology.md

```diff
@@ -134,3 +134,11 @@
 * [Valid](https://w3c.github.io/rdf-semantics/spec/#dfn-valid) – Any process which constructs a graph E from some other graph S is (simply) valid if S [simply entails](https://w3c.github.io/rdf-semantics/spec/#dfn-simply-entail) E in every case, otherwise invalid.
 * [Value space](https://w3c.github.io/rdf-semantics/spec/#dfn-value-space) – The value space of a datatype is the range of the [lexical-to-value mapping](https://w3c.github.io/rdf-concepts/spec/#dfn-lexical-to-value-mapping).
 * [Vocabulary](https://w3c.github.io/rdf-semantics/spec/#dfn-vocabulary) – A vocabulary is a set of [names](https://w3c.github.io/rdf-semantics/spec/#dfn-name).
+
+## New terms:
+
+* [Opaque semantics] – Any semantics for quoted triples where the syntactic form of the arguments to triples is important.  See [docs/semantics.md] for more information.
```

Comment:
These don't work as Markdown. The form `[Opaque semantics]` would need a URL to reference, (e.g., `[Opaque semantics](/docs/semantics.md#opaque-semantics)`). Same for `[docs/semantics.md]`, which might be `[Semantics](/docs/semantics.md)`.

Reply:
@gkellogg I think I have something that works now.